README.md

```diff
@@ -1,3 +1,171 @@
 # kube-service-bindings
 
-A repository with examples utilizing [kube-service-binding](https://www.npmjs.com/package/kube-service-bindings) npm package:
+A repository with examples utilizing [kube-service-bindings](https://www.npmjs.com/package/kube-service-bindings) npm package:
+
+- KAFKA
+  - node-rdkafka
+  - kafkajs
+- POSTGRESQL
+  - [pg](./src/pg/README.md)
+  - [odbc](./src/odbc-psql/README.md)
+- REDIS
+  - redis
+  - ioredis
+- MONGODB
+  - [mongoose](./src/mongoose/README.md)
+  - [mongodb](./src/mongodb/README.md)
+- AMQP
+  - [rhea](./src/rhea/README.md)
+- MYSQL
+  - [mysql2](./src/mysql2/README.md)
+  - [odbc](./src/odbc-mysql/README.md)
+  - [mysql](./src/mysql/README.md)
+
+# Basic instructions
+
```

Comment:
It would also be good to expand my suggestion that you need to install and then also deploy an instance. I think it would be good to keep those two steps together in the same section, possibly as sub-sections.

Reply:
More comments as I read further down.
I think in this section we should explain the general flow

1) isntall the general stuff
2) install the specific stuff needed for an exmple
3) deploy and run the example

If 2) is not listed in the example itself then having a table that shows which example needs what would be good.